.github/workflows/autobuild.yml

```diff
@@ -160,6 +237,7 @@ jobs:
             # Xcode 12.1.1 is the most-recent 12.1.x release:
             # https://xcodereleases.com/
             xcode_version:          12.1.1
+            is_main_build_target:   true
```

Comment:
It's non-optimal that we have to mark the Mac Legacy build as main build as well, but otherwise we would risk skipping CodeQL for Mac on many builds.

#2569 or some variation of that could help here.